### PR TITLE
Add KB article on migrating from KMFL to Keyman for Linux

### PR DESCRIPTION
**Pull request by darcywong00**
Addresses #94 on helping Linux users migrate from KMFL to Keyman for Linux

First commit was @ermshiperete 's answer on the [community site forum](https://community.software.sil.org/t/how-to-install-an-kmn-source-kmfl-file-onto-ubuntu-24-04/8864/5) (same response on Linux-user group.

Additional commits for formatting, and tweaking the section about a kmc compilation error.

### CODE REVIEW

**Comment by ermshiperete** [comment]
See also https://github.com/keymanapp/keyman/issues/11857

**Review by ermshiperete — APPROVED** [review]

**Comment by ermshiperete** [comment]
Just had a look at the KB page and noticed that the last paragraph ("To properly do this without an error, ...") is a bit misleading when it comes under the "Use Keyman for Linux to install the .kmp keyboard package" section header. We should either move that paragraph above the section header, or add a new section header above this paragraph.

File: knowledge-base/kb0114.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- Move your .kmn file to `source/mykeyboard.kmn`.
+
+- Build with:
+
+  ```bash
+  kmc build mykeyboard.kpj
+  ```
+
+Disregard the error: `"TypeError: Cannot read properties of undefined (reading 'description')"`, as the compiler will already have created the .kmp file in the `build`
+subdirectory.
```
Comment (mcdurdin) [inline knowledge-base/kb0114.md:69]: keymanapp/keyman#11856

File: knowledge-base/kb0114.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+```bash
+km-package-install -f build/mykeyboard.kmp
+```
+
+or with the Keyman Configuration tool.
+
+To properly do this without an error, you'll have to add some additional fields to the .kps file. It's easiest to let the `kmconvert` tool generate a template which will contain all necessary the fields:
+
+```bash
+kmconvert template -id mykeyboard
```
Comment (mcdurdin) [inline knowledge-base/kb0114.md:84]: Ref keymanapp/keyman#11014

**Pull request merged**